apps/new-tab/src/Settings.tsx

```
@@ -797,20 +1236,903 @@ function SettingsTrigger({
+                <div>
+                  <br />
+                  <TextFieldRoot class="mt-1 flex-1">
+                    <span class="text-sm font-medium text-muted-foreground">
+                      {chrome.i18n.getMessage("work_minutes")}
+                    </span>
+                    <br />
+                    <TextField
+                      placeholder={chrome.i18n.getMessage("work_minutes")}
+                      value={
+                        (typeof pomodoroConfig() === "object"
+                          ? (pomodoroConfig as Function)().workMinutes
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)